core/domain/topic_services.py

```diff
@@ -701,6 +704,26 @@ def add_additional_story(user_id, topic_id, story_id):
         'Added %s to additional story ids' % story_id)
 
 
+def rearrange_canonical_story(user_id, topic_id, from_index, to_index):
```

Comment:
Wait, this function is not needed, right? Wouldn't `apply_change_list` take care of this?

Reply:
Removed.